SEED Benchmark/OEI/classes/pollseedforproperties.xml

```diff
@@ -40,16 +43,18 @@ http://www.mulesoft.org/schema/mule/schedulers http://www.mulesoft.org/schema/mu
         <scripting:component doc:name="Groovy: write date processed to local file">
             <scripting:script engine="Groovy"><![CDATA[return new Date().format('yyyy-MM-dd HH:mm:ss.SSS');]]></scripting:script>
         </scripting:component>
-        <file:outbound-endpoint path="/tmp" outputPattern="lastReadDate.txt" responseTimeout="10000" doc:name="File"/>
+        <file:outbound-endpoint path="/tmp/oep" outputPattern="lastReadDate.txt" responseTimeout="10000" doc:name="File"/>

```

Comment:
Currently, this is where the lastReadDate.txt lives 

```
root@d04e46d6767e:/tmp/oep# ls
lastReadDate.txt
```

So was the code not working correctly before this change?

Reply:
Actually, I think this is just a copy of the other file that lives in the oei/pollseedforproperties.xml. So you are simply making the files consistent, correct?

Reply:
Correct, making sure all these are in agreement. This change was pulled in from master so this update is here to do that.